packages/owlbot-bootstrapper/cli/run-trigger.ts

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env node
+
+import {CloudBuildClient} from '@google-cloud/cloudbuild';
+import {logger} from 'gcf-utils';
+import yargs from 'yargs';
+
+async function runTrigger(argv: any) {
```

Comment:
This and `waitForBuild` look like common functionality between the CLI and bot, so it should probably be moved to `src/` and imported here. That will also make it easier to test.